eng/Generate.ps1

```diff
@@ -1,30 +1,40 @@
 #Requires -Version 6.0
-param($name, [switch]$noDebug, [switch]$reset)
+param($name, [switch]$noDebug, [switch]$reset, [switch]$noBuild, [switch]$fast)
 
 $ErrorActionPreference = 'Stop'
 
-function Invoke-AutoRest($autoRestArguments)
+function Invoke-AutoRest($baseOutput, $title, $autoRestArguments)
 {
-    $command = "npx autorest-beta $autoRestArguments"
+    $command = "npx autorest-beta $debugFlags $autoRestArguments"
+    if ($title)
+    {
+        $namespace = $title.Replace('-', '_')
+        $command = "$command --title=$title --namespace=$namespace"
+    }
+
+    if ($fast)
+    {
+        $codeModel = Join-Path $baseOutput $title "CodeModel.yaml"
+        $outputPath = Join-Path $baseOutput $title
+        $command = "dotnet run --project $autorestPluginProject --no-build -- --plugin=cs-modeler --title=$title --namespace=$namespace --standalone --input-file=$codeModel --output-path=$outputPath"
```

Comment:
Use $script:autorestPluginProject. And, there is an extra `--` in from of the `--plugin`.

Reply:
It's not extra, it's separator for arguments that are for `dotnet run` and the app it's invoking.

Reply:
That makes sense.